Gemfile

```diff
@@ -3,7 +3,7 @@ source 'https://rubygems.org'
 RAILS_VERSION = '~> 4.2.11'
 # RAILS_VERSION = '5.0.7.2'
 
-send :ruby, ENV['GEMFILE_RUBY_VERSION'] if ENV['GEMFILE_RUBY_VERSION']
+ruby File.read(".ruby-version")
```

Comment:
That old insanity was here due to an ancient issue with rvm:
https://github.com/errbit/errbit/issues/838

Do you know if it's been fixed sometime in the past 7 years?

Reply:
I will check and update PR if needed.

I look on rvm source code and I don't understand how it working under the hood. So, I will install VM and rvm inside to test.

Reply:
Ok. I am back. This year is crazy hard. It starts for me and my family in Mariupol, Ukraine. [The Washington Post on Mariupol news](https://www.washingtonpost.com/search/?query=mariupol&btn-search=). Running from war and writing open source it hard. So, I am trying. :)

So. I check rvm. And, let me describe what I found and think about all this.

Let's talk about `.ruby-version` file syntax and history. Everything start with [`rbenv`](rbenv/rbenv). `rbenv` is simple wrapper for bash to run ruby (and compile via `ruby-build`). Syntax is straightforward. e.g. `3.1.2`. Which tell rbenv wrapper to switch ruby version to `3.1.2`. Nothing more, nothing less.

Otherwise, rvm use (maybe, *uses in past*) `.rvmrc` which is bash script (this is way it hang terminal sometimes). And here starts all this voodoo-magic with parsing something in Gemfile, etc. rvm already understand `.ruby-version` syntax and just works. Even more, if rvm see `.ruby-version` file it will not try find ruby version anywhere more. And use ruby version from `.ruby-version` file. Just like `rbenv` do.

All ruby version managers now understand `.ruby-version` syntax and just switch ruby. This code is legacy and should be removed.

One more thing: I still not sure about ruby version (e.g. `ruby 3.1.2`) in Gemfile itself. Right now, I think, it should be. Looks like bundler ruby gems version solver use this information.

So, this is just legacy code that should be updated.